Source/UI/NewUI/BillingForm/Default Implementation/BillingForm/Header/Cancel/DefaultCancelButtonFormStyle.swift

```diff
@@ -1,15 +1,18 @@
 import UIKit
 
 struct DefaultCancelButtonFormStyle: ElementButtonStyle {
-
-    var text: String = "cancel".localized(forClass: CheckoutTheme.self)
+    var image: UIImage? = nil
```

Comment:
Do we need to assign a nil value its optional if anything not set it will be nil I suppose ?